addons/es/verifactu/bill.go

```diff
@@ -164,6 +172,17 @@ func validateInvoiceCustomer(val any) error {
 	)
 }
 
+func validateInvoiceCustomerSimplified(val any) error {
+	p, ok := val.(*org.Party)
+	if !ok || p == nil {
+		return nil
+	}
+	if p.TaxID != nil || len(p.Identities) > 0 {
+		return fmt.Errorf("F2 and R5 invoices cannot have a customer with tax ID or identities. If you want to fill customer details for a simplified invoice, use other type and add the extension %s", ExtKeySimplifiedArt7273)
+	}
```

Comment:
Would it not be easier to simply normalize this and always add the Simplified Article thing when the Tax ID or Identities are present?